libr/bin/bfile.c

```diff
@@ -876,36 +875,34 @@ R_API int r_bin_file_hash(RBin *bin, ut64 limit, const char *file) {
 	// By SLURP_LIMIT normally cannot compute ...
 	if (buf_len > limit) {
 		eprintf ("Cannot compute hash\n");
-		return -1;
-	}
-	//  XXX should use io api not raw file slurping
-	int blen = 0;
-	buf = (ut8*)r_file_slurp (file, &blen);
-	if (!buf) {
 		return false;
 	}
-	buf_len = blen;
-	if (!o->info) {
```

Comment:
This is if is necessary, otherwise it crashes when a rbin plugin doesn't have info implemented